CMakeLists.txt

```diff
@@ -339,6 +353,9 @@ STRING(REGEX REPLACE "\n" ";" FILELIST ${FILELIST})
 set(TARGET_NAMES "")
 # copy all correction files into the install location
 install(DIRECTORY data/ DESTINATION ${INSTALLDIR}/data)
+if (PAYLOADS)
+    install(DIRECTORY ${CMAKE_SOURCE_DIR}/analysis_configurations/${ANALYSIS}/payloads DESTINATION ${INSTALLDIR}/payloads)
```

Comment:
I got `build/bin/payloads/payloads/` on a test, I guess it's just `DESTINATION ${INSTALLDIR}/`?

Reply:
True, should be fixed now

Reply:
thanks, works fine

also, progress bar goes brrrrrrr